solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/List/ListView.swift

```diff
@@ -61,22 +64,29 @@ extension ListView: UITableViewDataSource {
   }
 
   public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-    let cell = tableView.dequeueReusableCell(withIdentifier: self.listViewCellIdentifier)!
-    cell.textLabel?.text = self.listItems[indexPath.row]
+       let cell = tableView.dequeueReusableCell(withIdentifier: self.listViewCellIdentifier) as! RepositoryCellView
+      cell.updateView(with: listItems[indexPath.row])
     return cell
   }
 }
 
+extension ListView: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 70
```

Comment:
In our case, this solution is just fine. However, what happens if a person sets the text size to 12 and decides to enable the zoom mode before using our app? The text will become huge but the cell separator will overlap the test.
Usually, we should rely on the system to calculate them for us. 

To do that, we need to set the `rowHeight` to `.automaticDimension` and we need to set the `estimatedRowHeight` to 70. [This article](https://medium.com/@vhart/self-sizing-tableview-cells-a165086d8631) explains the process. We should modify the lines where we create the `UITableView`.

Reply:
Nice! But I still maintain the extension to return the heightForRowAt or I remove it? When I changed to the estimatedRowHeight to 70 it started to crackle the cell

Reply:
Uhm... ok, I'll try this out.